src/services/users/userTypes.ts

```diff
@@ -0,0 +1,19 @@
+export interface createUserRequest {
+  user_signup_data: {
+    username: string,
+    password: string,
+    email: string,
+    first_name: string,
+    last_name: string,
+    phone_number: string,
+    country: string,
+    verified_legal_terms: boolean,
+  }
+}
+
+export interface authCodeRequest {
```

Comment:
Another interface copy

Reply:
Class name needs to user upper pascal case

Reply:
the correct interface is in `userRequestTypes.ts` file